src/test/kotlin/org/rust/lang/core/completion/RsAttributeCompletionTest.kt

```diff
@@ -272,6 +272,11 @@ class RsAttributeCompletionTest : RsAttributeCompletionTestBase() {
         fn foo() {}
     """)
 
+    fun `test no builtin attr completion for qualified path`() = checkNoCompletion("""
+        #[tokio::mai/*caret*/]
```

Comment:
It seems that `main` attribute is outdated and we should remove it from completion. So let's better test the `test` attribute